### PR TITLE
Add airbnb linter rules

### LINKED ISSUE #208: Add airbnb linter rules
**What would you like to be changed**:
Add `airbnb-base` to the eslint config.

**Why is this needed**:
It catches several existing errors that should accelerate future code review:

image

I've done initial config work in the branch `feat/add-airbnb-rules`, so all that is needed is to fix the linter errors it's catching. This is low priority, so I'd be willing to do it if the team is busy with other work.

### PR DESCRIPTION
**Pull request by pynnl**
Close #208

### CODE REVIEW

File: src/Session/Session.ts [hunk 1]
```diff
@@ -254,14 +260,16 @@ class Session {
...
     });
 
-    for (const capabilites of mergedCapabilities) {
-      const matchedCapabilities = Session.matchCapabilities(capabilites);
-      if (matchedCapabilities !== null) {
-        return matchedCapabilities;
-      }
-    }
+    let matchedCapabilities: Pluma.Capabilities | null = null;
+    mergedCapabilities.some(value => {
```
Comment (poftadeh) [inline src/Session/Session.ts:264]: I think semantically `filter` is more appropriate here. `some` I believe is more commonly used for its return value to check elements against a particular test function.

File: test/jest/CookieValidator.test.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const { CookieValidator } = require('../../build/Browser/CookieValidator');
```
Comment (poftadeh) [inline test/jest/CookieValidator.test.js:1]: The linter is currently ignoring the `test/` folder. To enable validation for test files we need to remove that exclusion from `.eslintignore` and set the env for jest globals in eslintrc:

 ```js
  "env": {
    "jest": true
  },
```
There are a bunch of linter errors in the test folder, so I'm not sure if you prefer to do that in another PR.

**Review by poftadeh — APPROVED** [review]
Looks good overall, thanks Tien. Some comments:

File: src/Session/Session.ts (hunk 1 above)
Comment (pynnl) [reply src/Session/Session.ts:264]: I believe `filter` would run for all items in the array, while `some` will stop when receive a `true`, which is what we want (taking the first matching capabilities).

Comment (poftadeh) [reply src/Session/Session.ts:264]: Oh that's right, you're looking for only the first match. But since you're storing the match, and not just checking if one exists, maybe `find` would be more apt. I'll leave it up to you to decide what's best.

Comment (pynnl) [reply src/Session/Session.ts:264]: `find` won't work because we want to store the result of `matchCapabilities`, not the items in the array. It's kinda like using `map` and `find` at the same time, but unfortunately JS doesn't support it yet 😢

File: test/jest/CookieValidator.test.js (hunk 2 above)
Comment (pynnl) [reply test/jest/CookieValidator.test.js:1]: Yes, I think it should be in another PR. I actually think of using `ts-jest` to write test in TS.

**Review by poftadeh — APPROVED** [review]

**Review by manekenpix — APPROVED** [review]

**Review by MeisterRed — APPROVED** [review]
